src/modules/videoconference/VideoConferenceProxyFilter/VideoConferenceProxyFilterx86.sln

```diff
@@ -7,6 +7,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "VideoConferenceProxyFilter"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "VideoConferenceShared", "..\VideoConferenceShared\VideoConferenceShared.vcxproj", "{459E0768-7EBD-4C41-BBA1-6DB3B3815E0A}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Version", "..\..\..\common\version\version.vcxproj", "{CC6E41AC-8174-4E8A-8D22-85DD7F4851DF}"
```

Comment:
worried that this may cause the version outputs to be double-struck during build; is that OK?

Reply:
It already is double-built today. The projects in this sln reference it so it definitely builds when this sln builds. The problem when it's not in the sln though is that the Configuration/Platform are lost, which means the 2nd build today is actually the wrong Configuration/Platform =P